src/ui/content-template/ContentTemplateSearchForm.js

```diff
@@ -44,6 +44,7 @@ class ContentTemplateSearchFormBody extends Component {
       onChangeSearchType,
       selectedAttribute,
     } = this.props;
+    console.log('selectedAtrribute', selectedAttribute);
```

Comment:
I guess this line was for debugging purpose and we can remove it ;)

Reply:
ups